gravitl/netclient
# fix(GRA-1387): remove join by sso/basic auth

TESTING STEPS:

Test server: https://api.nm.159-203-177-123.nip.io/

1. Open nc gui, chack that login by token, sso and basic auth are not available (enrollment key is th e only option)
2. Verify that join by enrollment key still works
